### DISCUSSION TITLE
Minecraft World Border to Display protected region

### DISCUSSION THREAD

**Discussion by wasder** [GriefPrevention/GriefPrevention#273] [Ideas]
Feature request: make protected regions displayed with minecraft's world border. https://minecraft.gamepedia.com/World_border

Would not it be cool?

**Comment by RoboMWM** [comment]
Not possible. This would:

- Only work for a single claim.
- Would prevent players from entering/exiting claim.
- Require ProtocolLib/Reflection/NMS to send fake worldborder packets.

There were some other suggestions to use structure blocks and the like though. #46

**Comment by wasder** [comment]
- Only display, when you check claim with stick/shovel
- Hide it when get too close to the border
- Make it as addon?

**Comment by bigpresh** [comment]
AFAIAA, can't be done - at least not without requiring a client-side addon, which would mean every player would have to have it.  The only way the server can make the client display the claim boundary as a world border would be by telling the client that that *was* the world border, and if it did that, you'd die if you were outside it.

Also, I'm pretty sure the world border is just implemented as distance from either 0,0 or the current spawn point, I forget which, so it wouldn't even work anyway.

Cheers for the suggestion though - I agree that, if it was possible to do it visually only without it actually being the world border and without a client-side addon then it'd be a nifty way to visualise claims, but I don't think it's doable.

**Comment by RoboMWM** [comment]
@bigpresh it's possible to use a fake worldborder to do all sorts of things (i.e. see the tinthealth project).

Sure, an addon would work with your given parameters.
